exotic/output_files.py

```diff
@@ -130,7 +134,9 @@ def aavso(self, comp_star, airmasses, ld0, ld1, ld2, ld3, epw_md5):
                 f.write(f"{round(self.fit.time[aavsoC], 8)},{round(self.fit.data[aavsoC], 7)},"
                         f"{round(self.fit.dataerr[aavsoC], 7)},{round(airmasses[aavsoC], 7)},"
                         f"{round(self.fit.airmass_model[aavsoC], 7)}\n")
-
+    def plate_status(self, plate_status: PlateStatus):
+        plate_status_file = self.dir / "temp" / f"PlateStatus_{self.p_dict['pName']}_{self.i_dict['date']}.csv"
+        plate_status.writePlateStatus(plate_status_file)
```

Comment:
It would be good to have a try-except here. Lots of things can happen during a write, like insufficient disk permissions for the user or an i/o failure.

Reply:
Would not presume to disagree, but it IS consistent with every other output file, in that none of them to do try-catch in the output method, and instead wrap them in a try-catch in the calling code in exotic.py - e.g. https://github.com/rzellem/EXOTIC/blob/bfaf8ddacf86cff3adb326126e219cb3e9ab2f05/exotic/exotic.py#L2533 vs https://github.com/rzellem/EXOTIC/blob/bfaf8ddacf86cff3adb326126e219cb3e9ab2f05/exotic/exotic.py#L2525 vs https://github.com/rzellem/EXOTIC/blob/bfaf8ddacf86cff3adb326126e219cb3e9ab2f05/exotic/exotic.py#L2519 and others.    Honestly, like the Pathlib comment below - as a PR, I was not intending to second guess the existing coding styles and conventions, whether or not I specifically agree with them - my goal with a PR is minimum impact versus being 'style police' :)

Reply:
OK ...